src/helpers/ruler.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * Add a horizontal ruler to your email
+ * 
+ * @example
+ * {{{ruler color="#ec6225" width="60px" height="3px" spacing="15px"}}}
+ * 
+ * Attribute Options
+ * color: Must be hex color value
+ * width: Can be percentage (50%) or pixel (50px) unit
+ * height: Defaults to pixel unit
+ * spacing: Default to pixel unit
+ * 
+ */
+
+module.exports = function(options) {
+
+	// Trim Non-Numberic Chracters
+	String.prototype.trimUnit = function() { return this.replace(/\D/g, ''); }
+	
+	// Variables & Options
```

Comment:
@IamManchanda thanks, I just checked preferences and tab_size is already set to 2 and translate tabs to spaces is set to true as well.

```
{
  // The number of spaces a tab is considered equal to
  "tab_size": 2,
  // Set to true to insert spaces when tab is pressed
  "translate_tabs_to_spaces": true,
  // Set to false to disable detection of tabs vs. spaces on load
  "detect_indentation": false,
}
```